stan-dev/math
# Feature/493 bernoulli logit glm lpmf

#### Submission Checklist


#### Summary:
Adds GLM with Bernoulli density and Logit link function.

#### Intended Effect:
Faster way of implementing said GLM, compared to using existing building blocks.

#### How to Verify:
Run tests to see it computes the same result as existing constructs.
We should also test for speed. I suspect the code can still be made much faster as I am new to C++.

#### Side Effects:
Extra vectorised test for the existing bernoulli_logit.

#### Documentation:

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):
Matthijs Vákár

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
